Composer/packages/lib/shared/src/path.ts

```diff
@@ -8,7 +8,7 @@ class PathHandler {
   resolve(...params: string[]) {
     const pathToTransform = path.join(...params);
     if (path.isAbsolute(pathToTransform)) {
-      return pathToTransform.replace(/\\/g, '/');
+      return pathToTransform.replace(/\\/g, '/')
```

Comment:
was this just to test the formatting?

Reply:
Yeah, but the annotations aren't working like I expected. They show up in the workflow details page, but not on the file.